matRad_calcParticleDose.m

```diff
@@ -161,6 +150,22 @@
     rot_coordsV(:,2) = rot_coordsV(:,2)-stf(i).sourcePoint_bev(2);
     rot_coordsV(:,3) = rot_coordsV(:,3)-stf(i).sourcePoint_bev(3);
     
+    % Calcualte radiological depth cube
+    lateralCutoff = 50;
+    fprintf(['matRad: Calculating radiological depth cube for beam ' num2str(i) ' ...']);
+    [radDepthCube,geoDistCube] = matRad_rayTracing(stf(i),ct,V,lateralCutoff);
+    geoDistBAMSCube = stf(i).BAMStoIsoDist - (stf.SAD - reshape(geoDistCube,size(ct.cube)));
+    stf(i).SSD = geoDistBAMSCube([stf(i).ray.ixSSD]);
```

Comment:
Maybe improve program design by replacing fokus vs. BAMS dist in base data by fokus vs SSD. That way it might easier for external users to understand what is going on here...